web/srv/api_handlers.go

```diff
@@ -434,11 +434,44 @@ func (h *handler) handleAPIResourceDefinition(w http.ResponseWriter, req *http.R
 	w.Write(resourceDefinition)
 }
 
-func (h *handler) handleGetExtension(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+func (h *handler) handleGetExtensions(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
 	ctx := req.Context()
 	extensionName := req.FormValue("extension_name")
 
 	resp := map[string]interface{}{}
+	if extensionName == "" {
+		extensions, err := h.k8sAPI.GetAllNamespacesWithExtensionLabel(ctx)
+		if err != nil {
+			renderJSONError(w, err, http.StatusInternalServerError)
+			return
+		}
+
+		type Extension struct {
+			Name string `json:"name"`
+			UID  string `json:"uid"`
+		}
+
+		extensionObject := Extension{}
+		extensionList := make([]Extension, 0, len(extensions))
+		if len(extensions) > 0 {
+			for _, extension := range extensions {
```

Comment:
this is a bit nitpicky but I think this block can be a bit confusing w.r.t to the data that's being copied. wdyt about renaming some of these variables to make it a bit clearer to understand what's going on?

for example (feel free to swap with your own)
```suggestion
			for _, installedExtension := range installedExtensions {
			  extension.Name = installedExtension.ObjectMeta.Name
```

It's pretty nitpicky so TIOLI :)